reana/reana_dev/git.py

```diff
@@ -820,4 +884,90 @@ def _push_to_origin(components):
         _push_to_origin(components)
 
 
+@click.option(
+    "--component",
+    "-c",
+    multiple=True,
+    default=["CLUSTER"],
+    help="Which components? [shortname|name|.|CLUSTER|ALL]",
+)
+@git_commands.command(name="git-create-release-commit")
+def git_create_release_commit_command(component):  # noqa: D301
+    """Create a release commit for the specified components.
+
+    \b
+    :param components: The option ``component`` can be repeated. The value may
+                       consist of:
+                         * (1) standard component name such as
+                               'reana-workflow-controller';
+                         * (2) short component name such as 'r-w-controller';
+                         * (3) special value '.' indicating component of the
+                               current working directory;
+                         * (4) special value 'CLUSTER' that will expand to
+                               cover all REANA cluster components [default];
+                         * (5) special value 'CLIENT' that will expand to
+                               cover all REANA client components;
+                         * (6) special value 'DEMO' that will expand
+                               to include several runable REANA demo examples;
+                         * (7) special value 'ALL' that will expand to include
+                               all REANA repositories.
+    :type component: str
+    """
+    for comp in select_components(component):
+        if git_create_release_commit(comp):
+            display_message("Release commit created.", comp)
+
+
+@git_commands.command(name="git-create-pr")
+def git_create_pr_command(component):  # noqa: D301
+    """Create a GitHub pull request for each selected component.
+
+    \b
+    :param components: The option ``component`` can be repeated. The value may
+                       consist of:
+                         * (1) standard component name such as
+                               'reana-workflow-controller';
+                         * (2) short component name such as 'r-w-controller';
+                         * (3) special value '.' indicating component of the
+                               current working directory;
+                         * (4) special value 'CLUSTER' that will expand to
+                               cover all REANA cluster components [default];
```

Comment:
Actually `default` is not `CLUSTER`, if you don't pass any `-c`, you get a traceback:

```console
$ reana-dev git-create-pr
...
TypeError: git_create_pr_command() missing 1 required positional argument: 'component'
```

Perhaps it is good not to introduce any default, but require users to always provide `-c` option, as this could be potentially disruptive command creating a plethora of PRs by mishap.

Ditto for other potentially disruptive commands such as `git-create-release-commit`, let's require `-c` for all those?